releases/__init__.py

```diff
@@ -598,8 +598,8 @@ def unknown_visit(self, node):
 
 def generate_changelog(app, doctree):
     # Don't scan/mutate documents that don't match the configured document name
-    # (which by default is changelog.rst).
-    if app.env.docname != app.config.releases_document_name:
+    # (which by default is ['changelog.rst', ]).
+    if app.env.docname not in app.config.releases_document_name:
```

Comment:
Sorry, it was off the cuff remark, I didn't check. The correct syntax based on that error message would be `.*/changelog.rst` rather than the glob syntax I used.

My point was that defining a list of files is cumbersome if you have a large project with an unknown set of changelogs. Regexes tend to be much more practical for such cases.